android/orblibrary/src/main/java/org/orbtv/orblibrary/OrbSession.java

```diff
@@ -13,9 +14,8 @@
 import java.nio.ByteBuffer;
 import java.util.List;
 
-class OrbSession implements IOrbSession {
+public class OrbSession implements IOrbSession {
```

Comment:
I found something interesting here.
We can now change OrbSession back to be not public, remove this line in MainActivity and OrbProvider as well
```
import org.orbtv.orblibrary.OrbSession;
```
since we put voice receiver in OrbProvider now.

Reply:
Finish the changes